### ISSUE TITLE
Feature request: Switch focus to tab in a different pane

### ISSUE THREAD

**Issue by jagged3dge** [js-padavan/atom-enhanced-tabs#13]
Thank you for this lovely plugin :+1: 

Since I'm used to working in a split-screen setting, it'd help a lot if it was possible to switch focus to a tab in another pane. Currently if I select a tab of a file (it's open in another pane) from the list, it opens the same file again in the current pane instead of switching focus to the already opened tab in the other pane.

I'm not sure if it's just me or if there's some conflict with another plugin on my end.

**Comment by clearmisp** [comment]
I agree, I also use two panes and switching between tabs in this setup is somewhat awkward.
